backend/src/main/java/com/ambacam/repository/OperateurRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.ambacam.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+
+import com.ambacam.model.Operateur;
+
+public interface OperateurRepository extends JpaRepository<Operateur, Long>, JpaSpecificationExecutor<Operateur> {
+
+	public List<Operateur> findByCreeParId(Long id);
```

Comment:
Per default, every element in an interface is public, so it's redundant to mark it as public.